predicators/spot_utils/perception/object_detection.py

```diff
@@ -399,22 +399,24 @@ def _get_pose_from_segmented_bounding_box(
     return world_frame_pose
 
 
-def get_grasp_pixel(rgbds: Dict[str, RGBDImageWithContext],
-                    artifacts: Dict[str, Any], object_id: ObjectDetectionID,
-                    camera_name: str) -> Tuple[int, int]:
+def get_grasp_pixel(rgbds: Dict[str,
+                                RGBDImageWithContext], artifacts: Dict[str,
+                                                                       Any],
+                    object_id: ObjectDetectionID, camera_name: str,
+                    rng: np.random.Generator) -> Tuple[int, int]:
     """Select a pixel for grasping in the given camera image."""
 
     if object_id in OBJECT_SPECIFIC_GRASP_SELECTORS:
         selector = OBJECT_SPECIFIC_GRASP_SELECTORS[object_id]
         return selector(rgbds, artifacts, camera_name)
 
-    return get_object_center_pixel_from_artifacts(artifacts, object_id,
-                                                  camera_name)
+    return get_random_mask_pixel_from_artifacts(artifacts, object_id,
+                                                camera_name, rng)
 
 
-def get_object_center_pixel_from_artifacts(
+def get_random_mask_pixel_from_artifacts(
```

Comment:
can you grep for calls to this function and make sure we're not leaving any dead code in the repo?